src/mig/modules/netstat/netstat_darwin.go

```diff
@@ -62,8 +64,104 @@ func HasSeenMac(val string) (found bool, elements []element, err error) {
 	return
 }
 
+func parseEndpointString(str string) (ip net.IP, port int) {
+	re, err := regexp.Compile("^(.*?)(%[a-z0-9]+)?\\.(\\*|[0-9]+)$")
+	if err != nil {
+		panic(err)
```

Comment:
I would recommend not panicking directly, but instead return an error value here, or panic and recover.